java/ql/test/utils/flowtestcasegenerator/pom.xml

```diff
@@ -0,0 +1,133 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                      http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.semmle</groupId>
+  <artifactId>parent</artifactId>
+  <version>1.0</version>
+  <repositories>
+    <repository>
+      <id>google</id>
+      <name>Google Maven</name>
+      <url>https://maven.google.com/</url>
+    </repository>
+  </repositories>
+  <properties>
+    <android.platform>31</android.platform>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+  <dependencies>
```

Comment:
Probably most of these dependencies aren't needed for this test case, but I get that it's a bit of a pain going through all of them to determine which ones can be removed 😄

Reply:
Yeah - you are right.
Note that this is not anything that is executed by the CI but merely some internal tooling for doing developer testing of the flow test case generator.
Thank you for the review!